code/modules/clothing/under/miscellaneous.dm

```diff
@@ -120,7 +120,7 @@
 	displays_id = 0
 
 /obj/item/clothing/under/rank/centcom/blueshield
-	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant\" and bears \"N.S.S. Cyberiad\" on the left shounder."
+	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant\"."

```

Comment:
Why the fuck did you change this?

Reply:
Because we are going for having more stations in the future to my knowledge, and that was the easiest way?

Reply:
I thought no one and their mother wanted more stations and thereby maps that would need to be maintained. Also not sure how it would work as we only use one map ever, but wouldn't they all be called the Cyberiad anyways?

Reply:
Still no reason to change

Reply:
No, they wouldn't be. Just look up the `using_map` datum, and technically we already have another station, which is Meta.